### DISCUSSION TITLE
Global Code Search on GitHub Mobile  🔍 🤝 📱

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#87452] [Mobile]
![Screenshot 2024-01-15 at 3 02 31 PM](https://github.com/community/community/assets/95834130/82d9aec1-07c6-41fd-8e68-964bdbf95e93)

### Introducing Global Code Search on GitHub Mobile

Global code search is now available directly from the home screen on GitHub Mobile. 🎉  This addition enables users to conveniently find code snippets, navigate repositories, and access content directly from the home screen.  

With global code search, users can easily locate anything they need while on the go.

**Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.** 🚀 

[Learn more about GitHub Mobile](https://github.com/mobile) and [share your feedback in the community](https://github.com/orgs/community/discussions/categories/mobile) to help us improve. 🤝

**Comment by ghost** [comment]
This is awesome finally mobile users search could be more accurate.

**Comment by CaioPaulinoCosta** [comment]
What a great addition! Having global code search directly on the home screen of GitHub Mobile makes life so much easier for developers. Now, quickly finding code snippets, navigating repositories, and accessing content on the go is a breeze. This addition undoubtedly enhances the power and convenience of GitHub Mobile. I'm already looking forward to exploring all the possibilities this feature offers. Congratulations to the GitHub team on yet another advancement! 🎉🚀

**Reply by ghost** [reply]
Indeed, the inclusion of global code search on GitHub Mobile is a significant convenience and productivity boost for developers.

**Comment by ghost** [comment]
Exciting to see the addition of Global Code Search on GitHub Mobile, enhancing collaboration and code exploration!

**Reply by CaioPaulinoCosta** [reply]
Indeed! The introduction of Global Code Search on GitHub Mobile is a thrilling development, amplifying collaboration opportunities and facilitating seamless code exploration!

**Comment by jonyahmed09123** [comment]
Ik
